tools/linter.py

```diff
@@ -14,6 +14,7 @@
 # computing the diff itself.
 EXCLUDE = (
     "numpy/typing/tests/data/",
+    "numpy/typing/_char_codes.py",
```

Comment:
In `_char_codes.py` I strongly prefer the longer line-length to keep things more ordered,
hence why it's been excluded from the linting tests. 

https://github.com/numpy/numpy/blob/1cadccc6012124e38e02a78b39bf268123d272c1/numpy/typing/_char_codes.py#L81-L95